jupyterhub/templates/scheduling/user-scheduler/deployment.yaml

```diff
@@ -47,7 +47,7 @@ spec:
       {{- end }}
       containers:
         - name: kube-scheduler
-          {{- if ge (atoi .Capabilities.KubeVersion.Minor) 21 }}
+          {{- if ge (atoi ( .Capabilities.KubeVersion.Minor | replace "+" "" )) 21 }}
```

Comment:
```suggestion
          {{- /*
            FIXME: We have added a workaround for EKS where
                  .Capabilities.KubeVersion.Minor can return a
                  string like "22+" instead of just "22".

                  See https://github.com/aws/eks-distro/issues/1128.
          */}}
          {{- if ge (atoi (.Capabilities.KubeVersion.Minor | replace "+" "")) 21 }}
```